packages/front-end/components/EventWebHooksPage/utils.ts

```diff
@@ -0,0 +1,32 @@
+import { NotificationEventName } from "back-end/src/events/base-types";
+
+export type EventWebHookEditParams = {
+  name: string;
+  url: string;
+  events: NotificationEventName[];
+};
+
+export const eventWebHookEventOptions: {
+  id: NotificationEventName;
+  name: NotificationEventName;
+}[] = [
+  {
+    id: "feature.updated",
+    name: "feature.updated",
+  },
+  {
+    id: "feature.created",
+    name: "feature.created",
+  },
+  {
+    id: "feature.deleted",
+    name: "feature.deleted",
+  },
```

Comment:
At launch, this list will be revised to include only the events we are supporting.